main.py

```diff
@@ -27,13 +47,32 @@ def get_todays_gazette():
         "Content-Type": "application/json",
         "Authorization": os.getenv("MARIA_QUITERIA_API_TOKEN"),
     }
-
-    response = requests.get(url, headers=headers, params=params)
-    response_json = response.json()
-
-    for result in response_json["results"]:
-        gazettes.append(result)
-
+    success = False
+    number_of_attempts = 0
+    while success == False and number_of_attempts < 4:
```

Comment:
Para comparar False, True ou None nós usamos o `is`.
```suggestion
    while success is False and number_of_attempts < 4:
```
Um link sobre isso: https://realpython.com/python-is-identity-vs-equality/